Resources/Prototypes/Entities/Clothing/Neck/scarfs.yml

```diff
@@ -8,6 +8,8 @@
     sprite: Clothing/Neck/Scarfs/red.rsi
   - type: Clothing
     sprite: Clothing/Neck/Scarfs/red.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
```

Comment:
Wouldn't this make the scarves as good at preventing chill as winter coats?

Reply:
I believe it would. I couldn't tell you how much a coefficient of 0.1 actually *does*, so I have no means of gauging what a meaningful amount would be beyond "does this protect against cold yes/no"

Reply:
It's worth seeing if higher coefficients would lead to something that's still tolerable and doesn't lead to _as quick of a frozen death_. Otherwise, if we make scarves as protective as winter coats, then scarves become the new best item to prevent cold damage, because they won't take up the armor slot, allowing people to wear both armor and frost protection instead of having to make a decision.

I believe scarves are also a mapper-only thing, right? They're not printable from the uniform printer, and they don't come in any cargo crates, do they? That would add rarity onto utility, making them even more desirable.

I'm imagining scarves should be about half as good as a winter coat. So assuming `0.1` means "completely resists normal Glacier temperature," `0.2` might be a good starting point. Some testing will be necessary to find the right number and rate of cold damage.

Reply:
Scarves are actually present in the Clothesmate.

Reply:
So they could actually be restocked through that absurdly expensive crate. Okay.

Reply:
It is worth mentioning that Winter Coats do offer 25% heat and 5% slash resists, so it isn't *necessarily* that scarves are the new meta, but you could certainly say having a scarf + winter coat would be the meta until you can find a kevlar vest or something with better armor properties than a winter coat.

I do agree that having them have *some* cold protection is most definitely worthwhile (as it is the reason we're doing this in the first place), so I'll go ahead and test what happens when adjusting the TemperatureProtection value a few times.

Reply:
So from my brief amount of testing: as the coefficient goes up, you receive **less** temperature protection.
0.1 is the normal for winter coats.
At 0.2, there is no noticeable difference.
At 0.3, after about 45-60 seconds, the "Too cold" alert displays. After staying in the -60 degree area for about 5 more minutes, you begin to take damage.
At 0.32, after about 45-60 seconds, the alert displays, then deteriorates over the course of 2 and a half minutes until you begin to take damage.
At 0.33, after about 30-45 seconds, the alert displays, then deteriorates over the course of about 1-2 minutes until you begin to take damage.
At 0.4, after about 15 seconds, the alert displays, then deteriorates over the course of 30-45 seconds until you begin to take damage.

Reply:
Yes, that's how coefficients work. The temperature is being modified by the number in the coefficient field via multiplication. So `x * y = z` where x is a temperature, y is the coefficient, and z is the result. It's just like armor.

Reply:
Either way, it looks like 0.3 is the sweet spot we can hit here to make scarves do something meaningful without being "on-par" with coats.

Do you think I should also change some of the other outer clothing to put more weight on "winter coats are best?" As far as hoodies and robes are concerned, they could probably be a 0.2 or 0.25 from what I'm guessing.

Reply:
So long as a regular non-armor item is equipped in the outer clothing slot, I see no problem with giving it a bit of temperature resistance when it makes sense, like robes and hoodies.

I like the idea of scarves alone being just good enough to get around in Glacier outside from one area to another in a few moments without taking damage, so regular outer clothing could be about that.

It's worth getting @OldDanceJacket to weigh in on this too, since this is tangentially related to maps and he could give us an idea of how cold Glacier is really supposed to be with regard to what sort of clothing you'd need.

We should standardize what values get what. Leaving winter coats as they are at 0.1 is probably fine. 0.3 or higher might be good for the previously-only-cosmetic items you mentioned, since wearing two items at 0.3 coefficient would be better than one winter coat in theory.

Reply:
Yeah, how you two are talking about the values is what I agree with. A winter coat should get you around just fine, but other, thinner items would require the inclusion of a scarf, or just not being out in the cold for too long.
We talked about what items should be "Glacier-proof" previously so we can go through it one more time and have the changes down.